proposals/lip-0028.md

```diff
@@ -31,37 +31,37 @@ When a transaction object is serialized, also the asset property has to be seria
 
 This proposal follows the specifications in the [LIP 0027](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0027.md) to define a generic and deterministic way of serializing transactions in the Lisk protocol. This implies that JSON schemas have to be defined in order to perform the serialization-deserialization process.
 
-As it is shown in the diagram below, every transaction object in the Lisk protocol, regardless of the type, has a common set of 6 properties. The data related to the specific transaction type is given in the `asset` property. This architecture is also expected to be used for custom transactions created with the Lisk SDK. For simplicity and consistency, custom transactions should have these 6 properties, while all the new logic and data is defined with respect to the custom `asset` property.
+As it is shown in the diagram below, every transaction object in the Lisk protocol, regardless of the type, has a common set of 7 properties. The data related to the specific transaction type is given in the `asset` property. This architecture is also expected to be used for custom transactions created with the Lisk SDK. For simplicity and consistency, custom transactions should have these 7 properties, while all the new logic and data is defined with respect to the custom `asset` property.
```

Comment:
```suggestion
As it is shown in the diagram below, all transaction objects in the Lisk protocol have a common set of 7 properties. The data related to the specific transaction type is given in the `asset` property. This architecture is also expected to be used for custom transactions created with the Lisk SDK. For simplicity and consistency, custom transactions should have these 7 properties, while all new data is defined as part of the custom `asset` property.
```

Reply:
I suggested to remove some mentions of "type" where it was not necessary. As this LIP only talks about the transaction data, I would remove "all the new logic". Also note that custom transaction logic can still uitlize base properties such as the sender.